bolt/src/test/java/test_locally/app/OAuthCallbacksTest.java

```diff
@@ -0,0 +1,24 @@
+package test_locally.app;
+
+import com.slack.api.bolt.App;
+import com.slack.api.bolt.AppConfig;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.Test;
+
+@Slf4j
+public class OAuthCallbacksTest {
+
+    @Test
+    public void persistenceCallback() {
+        App app = new App(AppConfig.builder()
+                .signingSecret("secret")
+                .clientId("111.222")
+                .clientSecret("secret")
+                .scope("commands,chat:write")
+                .build());
+        app.oauthPersistenceCallback(args -> args.getResponse().setStatusCode(200));
+        app.oauthPersistenceErrorCallback(args -> args.getResponse().setStatusCode(404));
+
+        // TODO: assertions
```

Comment:
Should we fill this test out? 😅 

For my own education: would this be done similarly to [the other OAuth test in this project](https://github.com/slackapi/java-slack-sdk/blob/main/bolt/src/test/java/test_locally/app/OAuthStartTest.java#L35), where we instantiate a new `OAuthStartRequest` and feed it into `app.run`?

Reply:
@filmaj 
>Should we fill this test out? 😅

Yes, we should. This commit just checks the successful compilation so far.

> where we instantiate a new OAuthStartRequest and feed it into app.run?

Similar, but the callback (= /slack/oauth_redirect) test is not that simple like /slack/install patterns. It requires more efforts (e.g., having mock server response, mock installation service) here.

Reply:
I've added the tests covering the patterns!